src/components/v5/shared/Navigation/ColonySwitcher/partials/ColonySwitcherAvatar.tsx

```diff
@@ -2,30 +2,19 @@ import clsx from 'clsx';
 import React from 'react';
 
 import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
-import { usePageLayoutContext } from '~context/PageLayoutContext/PageLayoutContext.ts';
-import { useTablet } from '~hooks';
 import ColonyIcon from '~icons/ColonyIcon.tsx';
 import Avatar from '~v5/shared/Avatar/index.ts';
 
-export const ColonySwitcherAvatar = () => {
+export const ColonySwitcherAvatar = ({ className }: { className?: string }) => {
```

Comment:
Just a personal preference thing here, but if this className is being passed just to differentiate between mobile and desktop, could we not just handle these class names inside the ColonySwitcherAvatar component itself? Otherwise it looks like I can style the ColonySwticherAvatar however I want, even though it should be quite specific and self contained.